lib/src/mt_dev.c

```diff
@@ -2234,7 +2234,7 @@ int mt_dev_if_init(struct mtl_main_impl* impl) {
         inf->max_tx_queues++; /* tap tx queue */
 #endif
       }
-      if (mt_no_system_rxq(impl) || mt_has_rss(impl, i) || mt_udp_transport(impl, i)) {
+      if (mt_no_system_rxq(impl) || mt_udp_transport(impl, i)) {
```

Comment:
@jianxind, what max_rx_queues value is suitable for rss path?

Reply:
Maybe "user_rx_queues_cnt * 2" to avoid queue conflict?